docs/packages/pkg/CppJwt.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    CppJwt
+
+.. index::
+  single: JWT ; CppJwt
+
+.. _pkg.CppJwt:
+
+CppJwt
+======
+
+-  `Official <https://github.com/QuoineFinancial/cpp-jwt>`__
+-  `Hunterized <https://github.com/QuoineFinancial/cpp-jwt>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/CppJwt/CMakeLists.txt>`__
+-  Added by `Richard Hodges <https://github.com/madmongo1>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
```

Comment:
Please update `N`:
- https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#pull-requests

Reply:
strange, I thought I did. It must have got lost in the many branches I had to make

Reply:
btw which branch should I update in order to re-push this change,and on which repo?

Reply:
This pull request origin is `QuoineFinancial:pr.CppJwt`, so if you want to update the pull request, you should send new commits there.